tests/Fixer/Casing/MagicConstantCasingFixerTest.php

```diff
@@ -93,6 +93,10 @@ public function provideFixCases()
      */
     public function testFix74($expected, $input = null)
     {
+        if (\PHP_VERSION_ID >= 80000) {
```

Comment:
Can't we just have more than one requires annotation. `@requires PHP < 8.0` and `@requires PHP 7.4`?

Reply:
I tried, but it doesn't work, might be worth to try to add to PHPUnit, but still we have to deal with now as well as I don't want to delay this work.